### PR TITLE
Testcases-of-plugin

### PR DESCRIPTION
**Pull request by adnan1naeem**
This pull request has Tescases of react-router-plugin
screenshot from 2017-11-21 19-23-09

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
-  "name": "bluerain-plugin-react-router",
+  "name": "@blueeast/bluerain-plugin-react-router",
   "version": "0.5.0",
   "description": "A BlueRain Plugin that adds cross platform routing via react-router v4",
   "keywords": [],
-  "main": "dist/bluerain-plugin-react-router.umd.js",
-  "module": "dist/bluerain-plugin-react-router.es5.js",
-  "typings": "dist/types/bluerain-plugin-react-router.d.ts",
+  "main": "dist/bluerain-plugin-react-router-to-typescript.umd.js",
```
Comment (artalat) [inline package.json]: why?

File: package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  "module": "dist/bluerain-plugin-react-router-to-typescript.es5.js",
+  "typings": "dist/types/bluerain-plugin-react-router-to-typescript.d.ts",
   "files": [
     "dist"
   ],
-  "author": "adnan1994 <adnan.naeem@blueeast.com>",
+  "author": "BlueEast Team <code@blueeast.com>",
   "repository": {
     "type": "git",
     "url": ""
```
Comment (artalat) [inline package.json]: add url

File: package.json [hunk 3]
```diff
@@ -104,7 +108,10 @@
     "validate-commit-msg": "^2.12.2"
   },
   "dependencies": {
-    "@blueeast/bluerain-os": "^0.5.0",
-    "react-router-redux": "^4.0.8"
+    "@blueeast/bluerain-os": "^0.4.0",
+    "history": "^4.7.2",
+    "react-router-dom": "^4.2.2",
+    "react-router-redux": "^5.0.0-alpha.8",
+    "react-transition-group": "2.3.0-beta.0"
```
Comment (artalat) [inline package.json]: why do we have this dependency?

**Comment by umairfarooq44** [comment]
All the tests has been passed with 100% coverage
screenshot from 2017-11-24 15-41-01

**Pull request merged**